utils/resources/resources.go

```diff
@@ -27,11 +27,14 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime/pkg/client"
 
 	"github.com/kiegroup/kogito-serverless-operator/container-builder/client"
+	"github.com/kiegroup/kogito-serverless-operator/utils"
 
 	v08 "github.com/kiegroup/kogito-serverless-operator/api/v1alpha08"
 	"github.com/kiegroup/kogito-serverless-operator/utils/kubernetes"
 )
 
+var builderResourcesPath = utils.GetEnv("BUILDER_RESOURCES_PATH", "/usr/local/etc/serverless-operator")
```

Comment:
This will be usefull especially in local testing

Reply:
are there any use cases?

Reply:
we need to document it as well, I guess

Reply:
Yes, if you need to execute the Operator locally in debug, you can use that ENV variable to a different folder on your machine (for example your project workspace) where those files are present.

Reply:
sounds good, can you please add this info in the README?